ui/src/main/java/org/jtaccuino/core/ui/JavaCellFactory.java

```diff
@@ -206,13 +212,29 @@ private JavaCellSkin(JavaCell javaCell) {
                         }
                     });
 
+            var outputNodes = javaCell.getCellData().getOutputData().stream().map(od
+                    -> od.mimeBundle().entrySet().stream().map(me
+                            -> switch (me.getKey()) {
+                case "image/png" ->
+                    new ImageView(new Image(new ByteArrayInputStream(Base64.getDecoder().decode(me.getValue()))));
+                case "text/plain" ->
+                    new Label(me.getValue());
```

Comment:
strictly speaking this case is not required as it will be handled by the default case